stable/atlantis/templates/statefulset.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "atlantis.fullname" . }}
```

Comment:
Also add the annotation option to the stateful set.  That will allow people to use kube2iam for AWS credentials.

Reply:
Done.